person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    true if is_of_age? || @parent_permission
+  end
+
+  private :is_of_age?
```

Comment:
- [ ] good job creating the Person class 👍 however in this line method `is_of_age?` is made private, however, the method created is called `of_age?`. in order to avoid this kind of typo's you can put all private methods under the private keyword instead.